documentation/modules/exploit/windows/local/user_screensaver_persistence.rb

```diff
@@ -0,0 +1,134 @@
+## Vulnerable Application
```

Comment:
Rename `documentation/modules/exploit/windows/local/user_screensaver_persistence.rb` to `.md`

Reply:
wow. i think i committed a cardinal sin for the docs guy